src/bot/Slackbot.py

```diff
@@ -15,24 +15,27 @@ def get_message():
         url = "https://slack.com/api/conversations.history"
         token = "aaaaaa"
         header = {
-            "Authorization":"Bearer {}".format(token)
+            "Authorization": "Bearer {}".format(token)
         }
         payload = {
-            "channel":"AAAAAAAA"
-            #"channel": self.CHANNEL
+            "channel": "AAAAAAAA"
+            # "channel": self.CHANNEL
         }
         message = requests.get(url, headers=header, params=payload)
         return "message"
 
-
-    def send_message_to_slander_api():
-
-        return "slander"
+    def send_message_to_slander_api(self,message):
+        url = 'http://3.143.237.69/kusorep/score/'
+        message = message
+        param = {'msg': message}
+        res = requests.get(url, params=param)
+        slander = res.json()
```

Comment:
The slander level can be obtained with the following code.
`slander = res.json()["body"]["kusoripu_score"]`

Reply:
Thanks! I edited it.